core/src/toga/icons.py

```diff
@@ -40,15 +40,15 @@ def __get__(self, obj, owner):
 class Icon:
     @cachedicon
     def TOGA_ICON(cls) -> Icon:
-        return Icon("resources/toga", system=True)
+        return Icon("toga", system=True)
 
     @cachedicon
     def DEFAULT_ICON(cls) -> Icon:
-        return Icon("resources/toga", system=True)
+        return Icon("toga", system=True)
 
     @cachedicon
     def OPTION_CONTAINER_DEFAULT_TAB_ICON(cls) -> Icon:
```

Comment:
These should have some docstrings, which should at least explain the distinction between TOGA_ICON and DEFAULT_ICON. If there isn't any distinction, maybe we should deprecate one of them.

Reply:
Agreed documenting these is desirable... is it possible? How do you docstring a constant?

Reply:
Hm, I see these aren't normal `@properties`, so I'm not sure how Sphinx would deal with them. Maybe it could be done by making `cachedicon` copy the `__doc__` attribute from the underlying method to itself. Or maybe we could use the `#:` syntax that works in Enums (e.g. in `toga/core/src/toga/constants/__init__.py`).

Reply:
Oh wait... these are properties... Forgot about that.